resources/app.yaml

```diff
@@ -22,7 +22,25 @@ hooks:
               - name: hook
                 image: quay.io/gravitational/debian-tall:0.0.1
                 command: ["/usr/local/bin/kubectl", "apply", "-f", "/var/lib/gravity/resources/resources.yaml"]
-
+  update:
+    spec:
+      apiVersion: batch/v1
+      kind: Job
+      metadata:
+        name: logging-app-update
+      spec:
+        parallelism: 1
+        completions: 1
+        activeDeadlineSeconds: 100
```

Comment:
update is successfull if script does not return error at any stage
freeze simply marks changeset as completed, with no further modifications allowed. 
these are not transactions, simply tracking changesets, changes are applied after each command
rig status will terminate on early success
it fails and dumps info to stdout
rig freeze will be only called on successfull update